README.md

```diff
@@ -17,23 +17,25 @@ We are explicitly soliciting contributors to maintain those integrations labelle
-| **Mosquitto**         | [Github: oqs-demos/mosquitto](mosquitto)       | [Dockerhub: openquantumsafe/mosquitto](https://hub.docker.com/repository/docker/openquantumsafe/mosquitto)               | unsupported
-| **Envoy**         | [Github: oqs-demos/envoy](envoy)       | [ Dockerhub: openquantumsafe/envoy](https://hub.docker.com/repository/docker/openquantumsafe/envoy)                       | unsupported
-| **Unbound**         | [Github: oqs-demos/unbound](unbound)       | [ Dockerhub: openquantumsafe/unbound](https://hub.docker.com/repository/docker/openquantumsafe/unbound)                       | unsupported
+|                   | **Build instructions**                                   | **Pre-built Docker image or binary files**                                                                                                                                                                                                  | Support? |
+|-------------------|----------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| -------- |
+| **curl**          | [Github: oqs-demos/curl](curl)                           | [Dockerhub: openquantumsafe/curl](https://hub.docker.com/repository/docker/openquantumsafe/curl), [Dockerhub: openquantumsafe/curl-quic](https://hub.docker.com/repository/docker/openquantumsafe/curl-quic)                                | @baentsch, @pi-314159
+| **Apache httpd**  | [Github: oqs-demos/httpd](httpd)                         | [Dockerhub: openquantumsafe/httpd](https://hub.docker.com/repository/docker/openquantumsafe/httpd)                                                                                                                                          | @baentsch
+| **nginx**         | [Github: oqs-demos/nginx](nginx)                         | [Dockerhub: openquantumsafe/nginx](https://hub.docker.com/repository/docker/openquantumsafe/nginx), [Dockerhub: openquantumsafe/nginx-quic](https://hub.docker.com/repository/docker/openquantumsafe/nginx-quic)                            | @baentsch, @bhess, @pi-314159
+| **Chromium**      | [Github: oqs-demos/chromium](chromium) (limited support) | -                                                                                                                                                                                                                                           | @pi-314159 |
+| **Locust**        | [Github: oqs-demos/locust](locust)                       | [ Dockerhub: openquantumsafe/locust](https://hub.docker.com/repository/docker/openquantumsafe/locust)                                                                                                                                       | @davidgca|
```

Comment:
Now reviewing all these lines one by one manually -- and that answers another of my questions: This _is_ targeted for publication on docker hub (which IMO is sensible) -- but then this also requires a (test-)build-and-push instruction in CI. Please add that, e.g., to [linux.yml](https://github.com/open-quantum-safe/oqs-demos/blob/main/.github/workflows/linux.yml)

Reply:
Sorry I'm not familiar to github workflow, and how can I test an eventually new CI,  this is mandatory? could we add docker image publication for next PR?

Reply:
Sure. @ajbozarth anyway volunteered to redo CI in this space afaik. But in that case, please remove the reference to the docker image at docker hub in this PR.

Reply:
remove it [here](https://github.com/open-quantum-safe/oqs-demos/pull/304/commits/ad093355501e576ec1eeffdf9e015781b0eb485e)
Thanks @baentsch  I want to complete this PR and this could maybe extend this PR unnecessary. I will take a look about your CI steps after merge this PR